test/functional/apps/console/ace/_autocomplete.ts

```diff
@@ -22,8 +22,6 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
     before(async () => {
       log.debug('navigateTo console');
       await PageObjects.common.navigateToApp('console');
-      // Ensure that the text area can be interacted with
-      await PageObjects.console.closeHelpIfExists();
```

Comment:
Removed usages of all test helpers for the old Help flyout since I added new ones.